### PR TITLE
First version of Doogie Mouse software setup

### PR DESCRIPTION
**Pull request by mateusmenezes95**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,382 @@
...
+
+## Finish the setup
+When you start your Raspberry Pi for the first time, some configuration should to be made. Follow the steps below to set up Raspberry:
+1. On the display upper left corner, click in the Raspberry ícon
+2. Go to `Preferences >> Raspberry PI Configuration` 
+3. On `System` window, change the fields:
+   - `Password` (current is raspberry). Recommended  `doogiemouse` for new password
+   - `Hostname`. Recommended `doogie-mouse`. 
+   **Note:** If multiples Doogie Mouse robots will be used, we recomend use a different second name, e.g., `doogie-brain` for the first robot and for the second robot, `doogie-brain`. Feel free to choose the best names...
+4. On `Interfaces`, enables SSH and I2C
```
Comment (caioaamaral) [inline README.md]: call the second  robot doogie-pinky

File: README.md [hunk 2]
```diff
@@ -1,2 +1,382 @@
...
+```
+
+### Instalation
+Now, we will download and build ROS Kinetic
+
+#### Create a catkin Workspace
+In order to build the core packages, you will need a catkin workspace. Create one now:
+```sh
+RPI:~$ mkdir -p ~/catkin_ws_isolated
+RPI:~$ cd ~/catkin_ws_isolated
```
Comment (caioaamaral) [inline README.md]: this workspace is jut for using in raspberry? Should be the same doogie_ws we are using to install doogie packages?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,382 @@
...
+
+## Connect your Raspberry Pi
+Make sure at least the keyboard and the monitor are connected to the Raspberry Pi. Now, power on the board using its power pins (5 V and GND) or the USB power port.
+
+## Finish the setup
+When you start your Raspberry Pi for the first time, some configuration should to be made. Follow the steps below to set up Raspberry:
+1. On the display upper left corner, click in the Raspberry ícon
+2. Go to `Preferences >> Raspberry PI Configuration` 
+3. On `System` window, change the fields:
+   - `Password` (current is raspberry). Recommended  `doogiemouse` for new password
```
Comment (caioaamaral) [inline README.md]: double spacement at 

> Recommended  `doogiemouse` for new password

File: README.md [hunk 4]
```diff
@@ -1,2 +1,382 @@
...
+6. Choose **Finish**
+
+### SSH using Linux or Mac OS
+You can use SSH to connect to your Raspberry Pi from a Linux computer, a Mac, or another Raspberry Pi, without installing additional software. You will need to know your Raspberry Pi's IP address to connect to it. To find this, type `hostname -I` from your Raspberry Pi terminal.
+
+To connect to your Pi from a different computer, copy and paste the following command into the terminal window but replace `<IP>` with the IP address of the Raspberry Pi. Use `Ctrl + Shift + V` to paste in the terminal.
+```sh
+HOST:~$ ssh pi@<IP>
+```
+When the connection works you will see a security/authenticity warning. Type  yes to continue. You will only see this warning the first time you connect.
```
Comment (caioaamaral) [inline README.md]: Double space at
> Type  yes to continue.

File: README.md (hunk 2 above)
Comment (mateusmenezes95) [reply README.md]: It is just to build ROS from the source. The `doogie_ws` will be used for the robot packages

**Pull request merged**